OWNERS_ALIASES

```diff
@@ -0,0 +1,4 @@
+aliases:
+  krew-index-maintainers:
```

Comment:
I think the `krew-maintainers` and `krew-index-maintainers` were swapped by accident. 

Probably this one should be `krew-maintainers` and this one [8888595ca25a7a48c303fbc07f369b2db9b95268/OWNERS_ALIASES#L2](https://github.com/GoogleContainerTools/krew-index/blob/8888595ca25a7a48c303fbc07f369b2db9b95268/OWNERS_ALIASES#L2) should be `krew-index-maintainers`

Reply:
Argh my bad. I think I’ll do both krew-maintainers if that makes it easier. I suspect we’ll revise them in the future again.

Reply:
At the moment, it doesn't really matter. The teams in the OWNERS files need to be defined in the OWNERS_ALIASES file _per repository_. Till the moment we can use GitHub teams for this, it is necessary to have the data available in the OWNERS* files for both repositories.

So it is completely up to you if you want to 'combine' or keep the repositories separate (different contributors/maintainers).